src/cogs/admin_commands.py

```diff
@@ -33,11 +33,11 @@ async def shutdown(self, ctx):
 
     @shutdown.error
     async def shutdown_error(self, ctx, error):
-        await ctx.channel.send("you're not an owner :cry:")
+        await ctx.channel.send(f"I'm sorry, {ctx.author}, I'm afraid I can't do that for you. You're not an owner :cry:")
```

Comment:
will this work correctly with the mention?

Reply:
Looks great, thanks :)

Reply:
To have it mentioned it should be `ctx.author.mention`